test/e2e/certificates_test.go

```diff
@@ -0,0 +1,102 @@
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/openshift/windows-machine-config-operator/controllers"
+	"github.com/openshift/windows-machine-config-operator/pkg/certificates"
```

Comment:
These should be in a separate import group

Reply:
Correct, I missed that.